api/apps/faq_app.py

```
@@ -0,0 +1,176 @@
+
+        return get_json_result(
+            data=data,
+            message="FAQ clusters fetched successfully",
+            code=settings.RetCode.SUCCESS,
+        )
+    except Exception as e:
+        logging.exception(e)
+        return server_error_response(e)
+
+    
+@manager.route("/fetch-cluster-by-id", methods=["GET"])
```

Ruff (0.8.2)

Undefined name `manager`

(F821)